src/complexity.cc

```diff
@@ -182,15 +182,15 @@ std::vector<BenchmarkReporter::Run> ComputeBigO(
     result_cpu = MinimalLeastSq(n, cpu_time, reports[0].complexity);
     result_real = MinimalLeastSq(n, real_time, result_cpu.complexity);
   }
-  std::string benchmark_name =
-      reports[0].benchmark_name.substr(0, reports[0].benchmark_name.find('/'));
+
+  std::string benchmark_name = reports[0].benchmark_name().substr(
```

Comment:
this should now be `run_name` right? at least, you assign it to `run_name` below.

Reply:
Hmm, right, sorry.
I think this `run_name` will be kinda broken for these range-based complexity measurements..